tests/queries/0_stateless/01926_bin_unbin.reference

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
In your previous version it works correctly, `bin(0)` should return  `'0'`. I'll fix it.